src/Microsoft.SqlTools.ServiceLayer/FileBrowser/FileBrowserOperation.cs

```diff
@@ -20,6 +21,9 @@ internal class FileBrowserOperation : FileBrowserBase
         private FileTree fileTree;
         private string expandPath;
         private string[] fileFilters;
+        private CancellationTokenSource cancelSource;
```

Comment:
>cancelSource [](start = 40, length = 12)

token source is disposable, make sure it gets disposed when the class is disposed